org-brain.el

```diff
@@ -3195,7 +3195,10 @@ LINK-TYPE will be \"brain\" by default."
               nil choice))))
     (let ((link (concat link-type ":"
                         (if (org-brain-filep choice) choice (nth 2 choice)))))
-      (push link org-link--insert-history)
+      (if (version-list-< (version-to-list (org-release)) '(9 3))
+        (push link org-insert-link-history)
+        (push link org-link--insert-history)
+        )
```

Comment:
The indentation here is a bit strange. I myself use `aggressive-indent-mode` when editing emacs lisp but you could also mark the region and use `M-x indent-region` (I think `TAB` does this by default too). Also parentheses shouldn't be on their own line in the emacs lisp code standard.

Reply:
Yeah thanks for the comment. I've made the changes.